src/components/screens/visitors-management-screen.tsx

```diff
@@ -522,14 +529,26 @@ export class VisitorsManagementScreen extends React.Component<Props, State> {
 
     this.setState({ contentLoading: true });
 
+    const visitorsApi = Api.getVisitorsApi(accessToken);
+    const createdVisitors: Visitor[] = [];
+    for await (const visitorToCreate of visitors.filter(_visitor => !_visitor.id)) {
```

Comment:
No need to await for since createVisitor is already awaited
```suggestion
    for (const visitorToCreate of visitors.filter(_visitor => !_visitor.id)) {
```